test/sharness/t0700-remotepin.sh

```diff
@@ -64,23 +102,24 @@ test_expect_success "output includes meaningful error" '
   test_cmp config_exp config_out
 '
 
-test_expect_success "'ipfs config show' doesn't include RemoteServices" '
-  ipfs config show > show_config &&
-  test_expect_code 1 grep RemoteServices show_config
+test_expect_success "'ipfs config show' does not include Pinning.RemoteServices[*].API.Key" '
+  ipfs config show | tee show_config | jq -r .Pinning.RemoteServices > remote_services &&
+  test_expect_code 1 grep \"Key\" remote_services &&
+  test_expect_code 1 grep fake_api_key show_config &&
+  test_expect_code 1 grep "$TEST_PIN_SVC_KEY" show_config
 '
 
-test_expect_success "'ipfs config replace' injects remote services back" '
-  test_expect_code 1 grep -q -E "test_.+_svc" show_config &&
+test_expect_success "'ipfs config replace' injects Pinning.RemoteServices[*].API.Key back" '
```

Comment:
WDYT about another test to make sure that we can change or add a policy in the config and the keys + endpoints stay the same?